Alertmanager: Add grafana config size limit (#9402)
grafana/mimir: CHANGELOG.md

  * `-query-scheduler.grpc-client-config.grpc-compression=s2`
  * `-ruler.client.grpc-compression=s2`
  * `-ruler.query-frontend.grpc-client-config.grpc-compression=s2`

> * [FEATURE] The following alertmanager limit was added: #9402
>   * User grafana config size (`-alertmanager.max-config-size-bytes`)

Reviewer: ```suggestion
* [FEATURE] Alertmanager: limit added for maximum size of the Grafana configuration blob (`-alertmanager.max-config-size-bytes`). #9402
```

   }
   ```
* [FEATURE] Added multi-zone ingesters and store-gateways support. #1352 #1552

> * [FEATURE] The following alertmanager limit was added: #9402

Reviewer: This needs to go at the top of the file (and not in the jsonnet section)

Author: lol... Tks, done now 👍
